modules/core/shared/src/main/scala/gem/enum/package.scala

```diff
@@ -14,7 +15,7 @@ import cats.implicits._
 /**
  * Enumerated types (normally generated from database tables) and related syntactic enrichments.
  */
-package object enum {
+package object enum extends ToPrismOps {
```

Comment:
This makes `Prism.unsafeGet` available to the generated enums. Alternatively we could have all the enum companions extend this trait. I don't have strong feelings.